### PR TITLE
Added docs from https://github.com/ElazarK/content-docs to content

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

Relates to demisto/etc#22874

### CODE REVIEW

**Comment by Itay4** [comment]
images should be under `docs`

**Comment by glicht** [comment]
There seem to be README files that don't have an associated palybook (or the README is not named correctly). For example:

https://github.com/demisto/content/blob/content_doc_migration/Playbooks/playbook-Prisma_Cloud_Compute_Cloud_Discovery_Alert_README.md

There should be a file named:

https://github.com/demisto/content/blob/content_doc_migration/Playbooks/playbook-Prisma_Cloud_Compute_Cloud_Discovery_Alert.yml but it is missing

**Comment by anara123** [comment]
This is how it looks in content repo
image

And this is how it looks in the website
image

File: Playbooks/playbook-Access_Investigation_Generic_NIST_README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+| DBotScore.Score | The indicator's score. | number |
+| IP | The IP address objects. | unknown |
+| Endpoint | The endpoint's object. | unknown |
+| Endpoint.Hostname | The hostname to enrich. | string |
+| Endpoint.OS | The endpoint OS. | string |
+| Endpoint.IP | The list of endpoint IP addresses. | unknown |
+| Endpoint.MAC | The list of endpoint MAC addresses. | unknown |
+| Endpoint.Domain | The endpoint domain name. | string |
+
+![Access_Investigation_Generic_NIST](https://github.com/demisto/content/blob/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Access_Investigation_Generic_NIST.png)
```
Comment (glicht) [inline Playbooks/playbook-Access_Investigation_Generic_NIST_README.md:72]: This link is wrong. It should point to the **raw** path:

https://raw.githubusercontent.com/demisto/content/1bdd5229392bd86f0cc58265a24df23ee3f7e662/docs/images/playbooks/Access_Investigation_Generic_NIST.png

Need to fix this in all playbook docs

**Comment by hod-alpert** [comment]
@anara123 Could you add a link to the specific file you saw it in?
Could not find it

**Comment by anara123** [comment]
@hod-alpert ignore my comment, seems that this file is not in your PR. Will have to fix it in different PR.

**Comment by glicht** [comment]
@hod-alpert Let's add a section `Playbook Image` before each image. So the image is displayed in it's own section. That way it will display in the table of contents and a user can jump to it directly. See my example commit at: https://github.com/demisto/content/blob/content_doc_migration/Playbooks/playbook-Autofocus_Query_Samples_and_Sessions_README.md

**Comment by glicht** [comment]
@kirbles19 are we ok to merge?

**Pull request merged**